src/patternfly/components/Wizard/docs/code.md

```diff
@@ -40,4 +40,3 @@
 | `.pf-m-no-padding` | `.pf-c-wizard__main` | Modifies the main container to remove the padding. |
 | `.pf-m-hover` | `.pf-c-wizard__nav-link` | Modifies a step link for the hovered state. This state is primarily for demonstration purposes and would not normally be used in lieu of the `:hover` pseudo-class. |
 | `.pf-m-focus` | `.pf-c-wizard__nav-link` | Modifies a step link for the focus state. This state is primarily for demonstration purposes and would not normally be used in lieu of the `:focus` pseudo-class.|
-| `.pf-m-align-right` | `.pf-c-wizard__footer` | Modifies the footer to right align it's content. |
```

Comment:
Can you also update the name of the second example to remove "right-aligned footer"

Reply:
yup! updated.